src/main/java/uk/gov/justice/hmpps/prison/service/PrisonerReleaseAndTransferService.java

```diff
@@ -674,18 +674,18 @@ private void generateAdmissionNote(final OffenderBooking booking, final AgencyLo
         caseNoteRepository.save(newCaseNote);
     }
 
-    private void incrementCurrentOccupancy(final AgencyInternalLocation assignedLivingUnit) {
+    private void decrementCurrentOccupancy(final AgencyInternalLocation assignedLivingUnit) {
         if (assignedLivingUnit != null) {
-            assignedLivingUnit.incrementCurrentOccupancy();
+            assignedLivingUnit.decrementCurrentOccupancy();
             agencyInternalLocationRepository.save(assignedLivingUnit);
-            incrementCurrentOccupancy(assignedLivingUnit.getParentLocation());
+            decrementCurrentOccupancy(assignedLivingUnit.getParentLocation());
         }
     }
 
 
     private void updateBeds(final OffenderBooking booking, final LocalDateTime releaseDateTime) {
```

Comment:
Not something you changed, so you don't have to change.... but the function name is too ambiguous - `deallocateBed` might be a better name